readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+Replace `707Andrew` with your Github username and copy the links to Pull Request description:
+- [DEMO LINK](https://707Andrew.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://707Andrew.github.io/layout_product-cards/report/html_report/)
```

Comment:
Don't forget to add this links in description

Reply:
Agree, add these links to the description